src/redisraft.h

```diff
@@ -682,6 +684,8 @@ typedef struct {
 #define CMD_SPEC_SORT_REPLY     (1 << 5) /* Command output should be sorted within a lua script */
 #define CMD_SPEC_RANDOM         (1 << 6) /* Commands that are always random */
 #define CMD_SPEC_DENYOOM        (1 << 7) /* Commands that can't be run when in oom */
+#define CMD_SPEC_FLAGS          (1 << 8) /* Commands that have script/function flags */
+#define CMD_SPEC_SEND_ACL       (1 << 9) /* Commands that have require we serialize ACL */
```

Comment:
I'd prefer calling this something like `CMD_SPEC_SCRIPT`. Sending ACL is what we will do to implement  raft logic correctly. I see cmdspec flags as the type of a command rather than the action we need to take.

Reply:
So, I sort of agree, but was wondering if it was correct.  it currently is, but if in the future we need some apply time acl checking, it wouldn't be.

Reply:
Apply time ACL checks for other commands? I think we can handle it by then if it becomes a problem. We can rename the flag or add another flag for those new commands. We don't have to maintain that behavior (deciding ACL serialization) under single flag anyway. 

So, until then, I feel like `CMD_SPEC_SCRIPT` is easier understand. Otherwise, whenever we see that flag in the commands.c, we'll have to jump to redisraft.c to see what does this flag actually mean.